viennacl-omp/pom.xml

```diff
@@ -133,15 +133,15 @@
             </goals>
             <configuration>
               <environmentVariables>
-                <LD_LIBRARY_PATH>{project.basedir}/target/classes/org/apache/mahout/javacpp/linalg/linux-x86_64_omp/
+                <LD_LIBRARY_PATH>{project.basedir}/target/classes/org/apache/mahout/javacpp/linalg/linux-x86_64/
```

Comment:
For now we can just leave this as a hardcoded hack, I think, but it would be good to have the directory read in a name from the properties file, and use that as the top level path name.